layouts/partials/content.html

```diff
@@ -50,4 +50,3 @@ <h2 id="available-scalers">
     </div>
   </div>
 </section>
-
```

Comment:
Nit: consider undoing this whitespace change since it's the only change in the file.

Reply:
Done!